README.md

```diff
@@ -55,6 +56,14 @@ This BudgetApp is a Ruby on Rails 7 project that helps you manage your expenses
 - [Click here](https://budget-app-5bnf.onrender.com/) to see the like demo!
```

Comment:
- [ ] Well done :clap: your project is very nice :+1: but your live version is not working when I click on the link you provided for the live demo it shows a `not found` error so kindly add the working link :slightly_smiling_face:

Reply:
You're right! I have updated the link. Thanks!